extensions/parquet/base/src/main/java/io/deephaven/parquet/base/ColumnChunkReader.java

```diff
@@ -69,4 +84,10 @@ public int getMaxId() {
     }
 
     PrimitiveType getType();
+
+    /**
+     * @return The "version" string from deephaven specific parquet metadata, or null if it's not present.
+     */
+    @Nullable
+    String getVersion();
```

Comment:
I tried passing TableInfo instead of version to ParquetFileReader class but that would require adding a dependency of Deephaven_Community_Core.extensions-parquet-base.main on Deephaven_Community_Core.extensions-parquet-table.main leading to circular dependency.

Reply:
OK